x-pack/platform/plugins/shared/ml/public/application/jobs/components/job_details_flyout/job_details_flyout.tsx

```diff
@@ -0,0 +1,147 @@
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { CombinedJobWithStats } from '../../../../../common/types/anomaly_detection_jobs';
+import { useMlApi, useMlLocator, useNavigateToPath } from '../../../contexts/kibana';
+import { JobDetails } from '../../jobs_list/components/job_details';
+import { loadFullJob } from '../../jobs_list/components/utils';
+import { useToastNotificationService } from '../../../services/toast_notification_service';
+import { ML_PAGES } from '../../../../../common/constants/locator';
+import { useJobInfoFlyouts } from './job_details_flyout_context';
+
+const doNothing = () => {};
+export const JobDetailsFlyout = () => {
+  const {
+    isDetailFlyoutOpen,
+    activeJobId: jobId,
+    setActiveJobId,
+    closeActiveFlyout,
+  } = useJobInfoFlyouts();
+  const flyoutTitleId = useGeneratedHtmlId({
+    prefix: 'jobDetailsFlyout',
+  });
+  const [isLoading, setIsLoading] = useState(false);
+  const [jobDetails, setJobDetails] = useState<CombinedJobWithStats | null>(null);
+  const mlApi = useMlApi();
+  const { displayErrorToast } = useToastNotificationService();
+
+  useEffect(() => {
+    let mounted = true;
+    const fetchJobDetails = async () => {
+      if (jobId) {
+        setIsLoading(true);
+        try {
+          const job = await loadFullJob(mlApi, jobId);
+          if (mounted && job) {
+            setJobDetails(job);
+          }
+        } catch (error) {
+          displayErrorToast(
+            error,
+            i18n.translate('xpack.ml.jobDetailsFlyout.errorFetchingJobDetails', {
+              defaultMessage: 'Error fetching job details',
+            })
+          );
+        } finally {
+          setIsLoading(false);
+        }
+      }
+    };
+    fetchJobDetails();
+    return () => {
+      mounted = false;
+    };
+  }, [jobId, mlApi, displayErrorToast]);
+
+  const navigateToPath = useNavigateToPath();
+  const mlLocator = useMlLocator();
+
+  if (!jobId) {
+    return null;
+  }
+
+  const openJobsList = async () => {
+    const pageState = { jobId };
+    if (mlLocator) {
+      const url = await mlLocator.getUrl({
+        page: ML_PAGES.ANOMALY_DETECTION_JOBS_MANAGE,
+        pageState,
+      });
+      await navigateToPath(url);
+    }
+  };
+
+  return isDetailFlyoutOpen ? (
+    <EuiFlyout
+      data-test-subj="jobDetailsFlyout"
+      type="overlay"
+      size="m"
+      ownFocus={false}
+      onClose={() => {
+        closeActiveFlyout();
+        setActiveJobId(null);
+      }}
+      aria-labelledby={flyoutTitleId}
+    >
+      <EuiFlyoutHeader hasBorder data-test-subj={`jobDetailsFlyout-${jobId}`}>
+        <EuiFlexGroup>
+          <EuiFlexItem grow={true}>
+            <EuiTitle size="s">
+              <h2 id={flyoutTitleId}>{jobId}</h2>
+            </EuiTitle>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty onClick={openJobsList} iconType="popout">
```

Comment:
Is the `popout` icon needed here as the jobs list is opening in the current page?

Reply:
Updated here [`466ddec` (#207141)](https://github.com/elastic/kibana/pull/207141/commits/466ddec987d94e12d4afef7df5b5f17fd4a85ea8)